servatrice/servatrice.sql

```diff
@@ -90,6 +90,7 @@ CREATE TABLE IF NOT EXISTS `cockatrice_rooms` (
   `name` varchar(50) NOT NULL,
   `descr` varchar(255) NOT NULL,
   `permissionlevel` varchar(20) NOT NULL,
+  `privlevel` enum("NONE","VIP","DONATOR") NOT NULL,
```

Comment:
I matched the wording for the column name in the users table in an attempt to keep confusion levels low.